langchain/package.json

```diff
@@ -218,4 +218,4 @@
       "import": "./document_loaders.js"
     }
   }
-}
```

Comment:
this is so annoying haha, yarn formats the `package.json` files with newline, something else formats them without